src/config/ipc.js

```diff
@@ -0,0 +1,26 @@
+const process = require('process')
+
+let ipcEnabled = false
+
+export const enableIpc = () => {
+  if (ipcEnabled === true) {
+    throw Error('IPC cannot be enabled twice')
+  }
+  ipcEnabled = true
+  listenForMessages()
+}
+
+const listenForMessages = () => {
+  process.on('message', m => {
+    console.log('got message', m)
+  })
+}
+
+export const notify = ({key, value}) =>{
+  if (ipcEnabled === true && process.send) {
```

Comment:
Curiosity: when could `process.send()` be `null`? (I'd also be happy with a "google is your friend" answer)